app/views/offer_mailer/new_offer.html.erb

```diff
@@ -0,0 +1,8 @@
+<p><%= @sender %> extended you <%= link_to "an offer", notification_url(@notification.record) %>:</p>
+
+<blockquote>
+  <p>Start date: <%= @offer.start_date %></p>
+  <p>Pay rate: <%= @offer.pay_rate_value %>/<%= @offer.pay_rate_time_unit_key %></p>
+  <p>Comment:</p>
+  <p><%= @offer.comment? ? @offer.comment : 'No comment' %></p>
```

Comment:
```suggestion
  <p><%= @offer.comment.presence || 'No comment' %></p>
```

I believe the `#comment?` doesn't add much, and that this could be on a decorator